src/Http/Permissions/GetPermissionsForCurrentUserRequestHandler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Permissions;
+
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Http\Response\PrivateJsonResponse;
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use CultuurNet\UDB3\Security\Permission\PermissionVoter;
+use CultuurNet\UDB3\Security\Permission\UserPermissionChecker;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+abstract class GetPermissionsForCurrentUserRequestHandler implements RequestHandlerInterface
+{
+    private UserPermissionChecker $userPermissionChecker;
+
+    private string $currentUserId;
+
+    /**
+     * @param Permission[] $permissions
+     */
+    public function __construct(
+        array $permissions,
+        PermissionVoter $permissionVoter,
+        string $currentUserId
+    ) {
+        $this->userPermissionChecker = new UserPermissionChecker($permissions, $permissionVoter);
+        $this->currentUserId = $currentUserId;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $routeParameters = new RouteParameters($request);
+        $offerId = $routeParameters->getOfferId();
```

Comment:
Now both offers and organizers work with an `offerId` route parameter. As discussed on Slack this would be better to be `organizerId` for the organizer endpoint. This could be done with an abstract method that calls `getOfferId` for offers and `getOrganizerId` for organizers.